src/sxastarter/src/components/play-travel/Accordion.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+  Box,
+} from '@chakra-ui/react';
+
+import {
+  ComponentParams,
+  ComponentRendering,
+  Text as JssTextField,
+  Image as JssImageField,
+  TextField,
+  RichTextField,
+  ImageField,
+} from '@sitecore-jss/sitecore-jss-nextjs';
+
+interface AccordionItemProps {
+  heading: { jsonValue: TextField };
+  content: { jsonValue: RichTextField };
+  image: { jsonValue: ImageField };
+}
+
+interface Fields {
+  data: {
+    accordion: {
+      children: {
+        results: AccordionItemProps[];
+      };
+    };
+  };
+}
+
+interface AccordionProps {
+  rendering: ComponentRendering & { params: ComponentParams };
+  params: ComponentParams;
+  fields: Fields;
+}
+
+const AccordionItemRendering = (props: AccordionItemProps) => (
+  <AccordionItem>
+    <h2>
+      <AccordionButton
```

Comment:
it could be just a line of code

<AccordionButton className={'h-20'} _hover={{ bgColor: 'white' }}>

Reply:
What do you mean?